### PR TITLE
env description in markdown file

### LINKED ISSUE #317: Document all env variable
Create a new md file (call it  Env.md) and document all environment variables that node uses

### PR DESCRIPTION
**Pull request by jamiehewitt15**
Fixes #317

Changes proposed in this PR:

- env description in markdown file

### CODE REVIEW

File: env.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+| NODE1_PRIVATE_KEY     | false    | "0xfd5c1ccea015b6d663618850824154a3b3fb2882c46cefb05b9a93fea8c3d215"                                                    | Used on test environments, specifically CI, represents the private key for node 1.                                                                |
+| NODE2_PRIVATE_KEY     | false    | "0x1263dc73bef43a9da06149c7e598f52025bf4027f1d6c13896b71e81bb9233fb"                                                    | Used on test environments, specifically CI, represents the private key for node 2.                                                                |
+| RPCS                  | false    | "{ \"1\":{ \"rpc\":\"https://rpc.eth.gateway.fm\", \"chainId\": 1, \"network\": \"mainet\", \"chunkSize\": 100 }, ...}" | JSON object defining RPC endpoints for various networks.                                                                                          |
```
Comment (bogdanfazakas) [inline env.md:9]: maybe we should have here a working RPCS env var with barge network or sepolia optimism

```suggestion
| RPCS                  | false    |"{ \"11155420\":{ \"rpc\":\"https://sepolia.optimism.io\", \"chainId\": 11155420, \"network\": \"optimism-sepolia\", \"chunkSize\": 1000 }}" | JSON object defining RPC endpoints for various networks.                                                                                          |
```

File: README.md [hunk 2]
```diff
@@ -80,13 +82,15 @@ For configuring the Indexer crawling interval in miliseconds (default, if not se
...
+### 6. Run the node
+
 Then start the node:
 
 ```bash
 npm run start
 ```
 
-### 6. Open a 2nd terminal and run another node
+### 7. Open a 2nd terminal and run another node
```
Comment (mariacarmina) [inline README.md:93]: I think here we can have sub-steps like for e.g. 6.1 and 6.2 for this section.

Comment (jamiehewitt15) [reply README.md:93]: That would make more sense in the docs where we would go into more detail and add sub-steps but for the main readme we're better off keeping it simple and at the same level. This is meant to be a quickstart guide to quickly get people up and running. 

I'd suggest we delete this step as it's the same as the previous step.

Comment (jamiehewitt15) [reply README.md]: I've replaced it with a note at the end saying "To run a second node, open a new terminal and follow these steps again."

**Review by paulo-ocean — APPROVED** [review]
lgtm, they are all there.
Not sure if its worth to mention somewhere, that all variables are also tracked on `src/utils/constants.ts` as well

**Comment by jamiehewitt15** [comment]
> lgtm, they are all there. Not sure if its worth to mention somewhere, that all variables are also tracked on `src/utils/constants.ts` : `ENVIRONMENT_VARIABLES` as well

Thanks for the review. Yeah fair point, I've added a short note about  `ENVIRONMENT_VARIABLES` within `src/utils/constants.ts`

**Pull request merged**